solid-ui.ttl

```diff
@@ -0,0 +1,771 @@
+         s:label "user prompt";
+         :prompt "A longer prompt for a user inputting this property" .
+    
+    :property     a r:Property;
+         s:comment """Many fields prompt for information about a given property of the subject.
+    When field is filled in, this gives which property is written into the data.""";
+         s:domain :Form;
+         s:label "property to be stored"@en;
+         s:range r:Property .
+    
+    :reference     a r:Property;
+         s:isDefinedBy <>;
+         s:label "Reference"@en,
+                "Referencia"@es .
+    
+    :required     a r:Property;
+         s:isDefinedBy <>;
+         s:label "Requerido"@es,
+                "Required"@en .
+    
+    :requiredError     a r:Property;
+         s:isDefinedBy <>;
+         s:label "Error requerido"@es,
+                "Required error"@en .
+    
+    :seqeunce     a r:Property;
+         s:comment "The sequence in which this item is arranged with repect to other parts.";
+         s:label "sequence number";
+         s:range xsd:integer .
+    
+    :size     a r:Property;
+         s:domain :ValueField;
+         s:label "size of field";
+         s:range xsd:integer;
+         :prompt "size of field in characters" .
+    
+    :sortBy     a r:Property;
+         s:comment """A property which typically is used to sort
+        members of a given class.""";
+         s:domain s:Class;
+         s:label "sort by";
+         s:range r:Property .
+    
+    :sortPriority     a r:Property,
+                owl:DatatypeProperty;
+         s:comment """When individuals or classes must be sorted, then
+        if they are given different values of sortPriority a user agent can
+        use this as a hint to how to present information.""";
+         s:label "sort priority";
+         s:range xsd:integer .
+    
+    :style     a r:Property,
+                owl:DatatypeProperty;
+         s:comment """Must be a valid CSS style string such as one could put in
+        an HTML style attribute.  Depending on the user interface system, this can
+        by given to individuals, classes or properties. It is up to a user interface
+        which wants to draw on them to pick how it uses styles from which parts
+        of the data it has.  For example, the style of a class may be picked
+        to distinguish information about things in that class.""";
+         s:label "style";
+         :prompt "CSS style" .
+    
+    :tableProperties     a r:Property;
+         s:comment """This  is a crude way of specifying a table-based
+            view for objects of this class.""";
+         s:domain s:Class;
+         s:label "table properties";
+         s:range r:List;
+         :prompt "Properties to be given in a default table view" .
+    
+    :use     a r:Property;
+         s:range :Form .
+    
+    :valid     a r:Property;
+         s:isDefinedBy <>;
+         s:label "Valid"@en,
+                "VÃ¡lida"@es .
```

Comment:
There are a number of capitalized and accented letters in the above, in the Spanish (Espanol) langstrings, which do not appear to be correct (but this _might_ be an issue between my Chrome browser and GitHub). I flag it so it may be investigated and corrected if appropriate.